scripts/install/linux_optional_compilation_dependencies.sh

```diff
@@ -9,21 +9,19 @@ fi
 apt install --yes software-properties-common
 add-apt-repository -y ppa:deadsnakes/ppa
 apt update
-apt install --yes lsb-release wget python3.5-dev python3.6-dev python3.7-dev python3.8-dev python3.9-dev
+apt install --yes lsb-release curl python3.6-dev python3.7-dev python3.8-dev python3.9-dev dirmngr
+curl -sL https://deb.nodesource.com/setup_15.x | sudo -E bash -
```

Comment:
Is the sudo really needed here (two occurrences)? It seems the script need to be run as sudoer anyhow because of the apt install commands.

Reply:
Yes, you are right. We can get rid of the sudo.